model/version_db.go

```diff
@@ -314,6 +314,17 @@ func VersionUpdateOne(query interface{}, update interface{}) error {
 	)
 }
 
+func UpdateVersionMessage(versionId, message string) error {
```

Comment:
in the comment on the ticket, it says that the problem is  `that the version gets the update but the patch does not.`. Is it really the opposite, that the patch gets it but not the version?

Reply:
that would make sense since it shows up on "my patches" but not the actual patch page.

Reply:
yeah, the patch in the db is being updated but the UI uses the version page displays the version message which isnt being updated